frontend/src/components/forms/CreateLog.tsx

```diff
@@ -192,59 +256,47 @@ const CreateLog = () => {
   };
 
   const handleSubmit = () => {
-    // log the state values for testing
-    // console.log({
-    //     employee,
-    //     date,
-    //     time,
-    //     building,
-    //     resident,
-    //     tags,
-    //     attnTo,
-    //     notes,
-    //     flagged,
-    // });
-
     // Update error states
-    setEmployeeError(employee === "");
+    setEmployeeError(employee.label === "" && employee.value === -1);
     setDateError(date === null);
     setTimeError(time === "");
     setBuildingError(building === "");
-    setResidentError(resident === "");
+    setResidentError(resident === -1);
     setNotesError(notes === "");
 
-    // log the error state values for testing
-    // console.log({
-    //     employeeError: employee === "",
-    //     dateError: date === null,
-    //     timeError: time === "",
-    //     buildingError: building === "",
-    //     residentError: resident === "",
-    //     notesError: notes === "",
-    // });
-
     // If any required fields are empty, prevent form submission
     if (
-      employee === "" ||
+      (employee.label === "" && employee.value === -1) ||
       date === null ||
       time === "" ||
       building === "" ||
-      resident === "" ||
+      resident === -1 ||
       notes === ""
     ) {
       return;
     }
 
     // Create a log in the db with this data
     setCreateOpen(false);
-    setShowAlert(true);
     // update the table with the new log
+    LogRecordAPIClient.createLog(employee.value, resident, flagged, notes, attnTo, building).then((res) => {
+      if (res != null) {
+        setAlertData(ALERT_DATA.SUCCESS)
+        getRecords(1);
+        setUserPageNum(1)
```

Comment:
Nitpick but I would switch `setUserPageNum` and `getRecords` to keep things consistent